### ISSUE TITLE
Badge printing templates per printer

### ISSUE THREAD

**Issue by jashort** [kumoregdev/kumoreg#7]
Printers, being terrible, sometimes need a little tweak one way or another to make everything line up. Especially since we'll likely have multiple models of printer in use. 

It would be neat if it were possible to either load a different template per printer, or set an offset for the template per printer, or both. Thoughts:
- Being able to change things on the fly (without restarting the software) is a must
- Format should be easy to change since the badge layout changes every year
- Needs to handle different fonts, colors, drawing primitives, and positioning
- Being able to easily change the layout as a whole for all the printers when the badge changes would be good, but should that wipe out all the tweaks? Definitely need to be able to change one printer without affecting others as well.

Basically, there's still some work to flesh out just how this would function.

**Comment by jashort** [comment]
Added a configurable offset in Computer records rather than a full template system. While it would still be nice to have full templates, I couldn't think of a reason that you would want completely different badge templates on different computers/printers.

The downside to this is that the initial badge layout will still be cumbersome.

This does satisfy the "able to change things on the fly" which is the bulk of the time-sensitive tweaking that will need to be done.
